tests/e2e/features/artifacts/bridgePage/artifacts.feature

```diff
@@ -4,6 +4,24 @@ Feature: Withdraw
   Background:
     Given Connect Metamask extension with login action
 
+  @id1609 @id1607
+  Scenario: Check Explore ecosystem button on Bridge Deposit
+    Given I go to page "/bridge?network=era-goerli"
+    When I click by text "Deposit"
+    When I choose "ETH" as token and insert "0.0000000001" as amount
+    When I "confirm" transaction after clicking "Add funds to zkSync Era Testnet" button
+    Then Message "Transaction submitted" should be visible
+    Then Element with "text" " Explore ecosystem " should be "visible"
+    Then Element with "href" "https://ecosystem.zksync.io" should be "visible"
+        #id1607 Check other elements on window
```

Comment:
Partly duplicate of https://allure.matterlabs.dev/project/17/test-cases/1404?treeId=30 . it is better to have a reference in the allure

Reply:
Steps to resolve the comment:

- removed from GitHub description
- removed from the initial Linear ticket
- added information in the Allure test cases with reference to the automated test case (in another PR)
- removed code
@pcheremu